static/extensions/MubiLop/penguinhook.js

```diff
@@ -0,0 +1,258 @@
+// Penguin Hook
+// By MubiLop from Ruby Team (https://rubyteam.tech/)
+// I didn't see any extension with embed support so yeah i made it.
+
+(function (Scratch) {
+
+    class MubilopPenguinHook {
+        'use strict'
+        embedProperties = {};
+
+        getInfo() {
+            return {
+                id: 'penguinhook',
+                name: 'PenguinHook',
+                color1: "#02b4e0",
+                color2: "#02b4e0",
```

Comment:
If all colors are the same then you shouldnt be defining past color1 or itll just look bad